compiler/rustc_codegen_cranelift/src/abi/mod.rs

```diff
@@ -22,7 +22,7 @@ fn clif_sig_from_fn_abi<'tcx>(
     fn_abi: &FnAbi<'tcx, Ty<'tcx>>,
 ) -> Signature {
     let call_conv = match fn_abi.conv {
-        Conv::Rust | Conv::C => default_call_conv,
+        Conv::Rust | Conv::C | Conv::RustCold => default_call_conv,
```

Comment:
You could use `CallConv::Cold` here in principle. Cranelift currently implements it the same as SystemV currently though and I'm not sure how well it handles mixed calling conventions.

Reply:
Oh, thanks! [Smallest caller code size](https://docs.rs/cranelift-codegen/latest/cranelift_codegen/isa/enum.CallConv.html#variant.Cold) sounds great; I should have looked earlier.